src/components/entity/ha-entity-state-picker.ts

```diff
@@ -44,10 +46,16 @@ class HaEntityStatePicker extends LitElement {
   }
 
   protected updated(changedProps: PropertyValues) {
-    if (changedProps.has("_opened") && this._opened) {
```

Comment:
Don't want to wait for combobox to be opened to populate the items list, because this list is what allows translation of states from raw value to pretty frontend value. 

In the current release, when you first open the editor, combobox text fields are populated with their raw backend values (e.g. "off"), but as soon as you click on the dropdown, it morphs to the pretty value ("Off"). This gives a more consistent behavior and shows the pretty translated value from the outset.